sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_patch.py

```diff
@@ -89,19 +103,15 @@ def __init__(
         # For ConfidentialLedgerCertificateCredential, pass the path to the certificate down to the
         # PipelineCLient.
         if isinstance(credential, ConfidentialLedgerCertificateCredential):
-            kwargs["connection_cert"] = kwargs.get("connection_cert", credential.certificate_path)
```

Comment:
Yeah, I don't think there's a point in using a kwarg when the customer usually passes a `ConfidentialLedgerCertificateCredential` (it's not like there's a default to be overridden, the customer would be overriding the credential they provide with the kwarg), and all our samples instruct customers to pass the certificate as the `ConfidentialLedgerCertificateCredential` type instead of as a kwarg.

I suppose this can be considered a breaking change, so please let me know if I should change it back :smile: But the only way a difference would be observed if a customer is providing both `ConfidentialLedgerCertificateCredential` and the kwarg, which doesn't seem likely.

Reply:
What would have happened if someone had passed both?
If it was functional, then I would probably leave as-is. Using kwargs as overrides, even to other objects they have passed in, is a pretty common pattern in our SDKs.

Reply:
Sure, I'll change it back then!